dast/deploy_ssml_api.sh

```diff
@@ -64,10 +64,12 @@ for api_doc in ${API_URL_LIST}; do
   oc delete pvc rapidast-pvc
   (( counter++ ))
 done
-
 python find_alert_types.py
-
+high_alert_status=$(echo $?)
+echo "high alert return status $high_alert_status"
 if [ $phase != "Succeeded" ]; then
```

Comment:
@paigerube14 Does it make sense to call find_alert_types.py to process results only if phase == succeeded.

Reply:
Not necessarily, if there are multiple apis defined the $phase would only be for the last one so we would still want totry to get the alerts from the others